patches/chrome-BUILD.gn.patch

```diff
@@ -120,31 +124,45 @@ index 960921acd7ebb0ba99092745e8d30bca76821f2a..534fe8a987d939b2dc24c8cf20d4e950
        "app/chrome_main_mac.h",
        "app/chrome_main_mac.mm",
      ]
-@@ -1226,6 +1268,7 @@ if (is_win) {
+@@ -1226,6 +1272,7 @@ if (is_win) {
  
      public_deps = [
        ":chrome_dll",
 +      "//brave/vendor/brave-extension:copy-extension-localization",
      ]
  
      deps = [
-@@ -1400,6 +1443,7 @@ if (is_win) {
+@@ -1253,6 +1300,13 @@ if (is_win) {
+       "-Wl,-order_file," + rebase_path("app/framework.order", root_build_dir),
+     ]
+ 
++    if (brave_chromium_build) {
```

Comment:
Do these need to be patches? Seems like they could go in a brave target

Reply:
I think we can probably get rid of all of these new patches for chrome/BUILD.gn and move them up to brave

Reply:
I tried but it's difficult w/o changing chrome/BUILD.gn.
`//brave/browser` config are propagated to `chrome_dll` target.
However, they are not propagated to `chrome_framework` because `chrome_framework` includes `chrome_dll` in deps(to get them public_deps should be used).

If `chrome_framework` inserts `chrome_dll` in `public_deps`, its configuration is propagated to `chrome_app`.

I did some cleanup with https://github.com/brave/Sparkle/pull/2. PTAL.